portfolio/src/main/webapp/js/script.js

```diff
@@ -41,10 +41,10 @@ function switchIcon(icon) {
 }
 
 /**
- * Display text from data Servlet
+ * get Number of comments from data servlet
```

Comment:
It's a small nit, but please pay attention to capitalization. `get Number of comments` -> `Get number of comments`